### PR TITLE
Added migration script

### PR DESCRIPTION
**Pull request by cbron**

### CODE REVIEW

File: istio-onefive-migration.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- name: default-user
+  user:
+    token: ${token}
+" > sa.kubeconfig
+kubectl config --kubeconfig=sa.kubeconfig use-context default-context
+
+versions=$(kubectl get --ignore-not-found=true deploy istio-galley -n istio-system  -o=jsonpath='{$.spec.template.spec.containers[*].image}')
+if [[ $versions == *"1.4"* || $versions == *"1.3"* ]]; then
+	echo "Preparing for migration to Istio 1.5"
+	kubectl delete --wait=true --timeout=20s --ignore-not-found=true deployment istio-galley -n istio-system
```
Comment (StrongMonkey) [inline istio-onefive-migration.sh:34]: Are we not taking the configmap approach to delete validatingwebhook?

Comment (cbron) [reply istio-onefive-migration.sh:34]: Removing webhook data from configMap does not affect existing webhook. Didn't dig into why, but no logs in galley suggest it's watching that file or erroring out along the way. Must be configured at install time only. I can show you setup.

**Review by brendarearden — APPROVED** [review]
LGTM

File: istio-onefive-migration.sh (hunk 1 above)
Comment (StrongMonkey) [reply istio-onefive-migration.sh:34]: ok, I guess it is only installed during startup. I think that's fine then.

**Comment by cbron** [comment]
Added drone file, ready for final review

**Review by brendarearden — APPROVED** [review]

**Review by StrongMonkey — APPROVED** [review]
LGTM :+1:

**Review by brendarearden — APPROVED** [review]
🔥

**Pull request merged**